packages/pg-pool/index.js

```diff
@@ -414,6 +439,10 @@ class Pool extends EventEmitter {
     return this._idle.length
   }
 
+  get expiredCount() {
```

Comment:
Is this mainly for the test, or do you see it being useful as a public API?

Reply:
It's useful as a public API to poll the count of busy connections pending expiration. For example, if you have some long running queries that are preventing expiration you could see that occurring. It is also useful for testing.